client/src/util/fetchFromAPI.test.js

```diff
@@ -1,6 +1,16 @@
 import { expect, test } from "@jest/globals";
 import { queryAPI } from "./fetchFromAPI";
+//reset the fetch mock so the previous tests fon't interfere with current tests
+beforeEach(() => {
+  fetch.resetMocks();
+});
 
+test("mocks the fetch request", async ({ options }) => {
```

Comment:
Hint: remove `options` from here.  It's not passed into the test.

```suggestion
test("mocks the fetch request", async () => {
```